results/cmd/api/main.go

```diff
@@ -57,7 +59,24 @@ type server struct {
 	db *sql.DB
 }
 
-func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.TaskRun, error) {
-	// TODO: implement this.
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTaskRun not implemented")
+// Receive TaskRun from Watcher and save it to local Sqlite Server
+func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.CreateTaskRunResponse, error) {
+	database := s.db
+	statement, _ := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid INTEGER PRIMARY KEY, taskrunlog BLOB)")
+	statement.Exec()
+	statement, _ = database.Prepare("INSERT INTO taskrun (taskrunlog) VALUES (?)")
+	blobData, _ := proto.Marshal(req.GetTaskRun())
+	statement.Exec(blobData)
+	rows, _ := database.Query("SELECT logid, taskrunlog FROM taskrun")
+	var logid int
+	var taskrunblob []byte
+	for rows.Next() {
+		rows.Scan(&logid, &taskrunblob)
+		taskrun := &pb.TaskRun{}
+		proto.Unmarshal(taskrunblob, taskrun)
+		fmt.Println(strconv.Itoa(logid) + taskrun.String())
+	}
+	statement, _ = database.Prepare("DROP TABLE taskrun")
```

Comment:
We shouldn't drop the table - won't this delete all the data? 🙃

Reply:
Oh, I leave it here because to do cleaning before the method worked, definitely would remove this line later